public/js/actions/sources.js

```diff
@@ -95,29 +86,30 @@ function newSources(sources) {
  * @static
  */
 function loadSourceMap(generatedSource) {
-  return ({ dispatch, getState }) => {
-    let sourceMap = getSourceMap(getState(), generatedSource.id);
-    if (sourceMap) {
+  return async function({ dispatch, getState }) {
```

Comment:
I like this style better than the `[PROMISE]` dispatch, in this and several cases. Thanks for cleaning this up.